ceph_bootstrap/deploy.py

```diff
@@ -1242,7 +1242,7 @@ def check_deploy_prerequesites(minion_id):
         host_ls = []
         for minion, value in result.items():
             if value:
-                host_ls = SaltClient.local().cmd(minion, 'ceph_orch.host_ls')[minion]
+                host_ls = SaltClient.local_cmd(minion, 'ceph_orch.host_ls').get(minion, [])
```

Comment:
Simply calling `local_cmd` instead of `local().cmd` is not equivalent ATM.
To fix this, you have to change the default value `tgt_type` value on `local_cmd` to `glob`:
```
def local_cmd(cls, target, func, args=None, tgt_type='glob'):
```

Reply:
fixed